app/models/cars.py

```diff
@@ -0,0 +1,17 @@
+from db import Base
+import sqlalchemy as sa
+
+
+class Car(Base):
+    __tablename__ = 'cars'
+
+    license_number = sa.Column(sa.Integer, primary_key=True)
+    brand_name = sa.Column(sa.String(45))
```

Comment:
lägg till nullable(not null), exempel,  brand_name = sa.Column(sa.String(45), nullable=False)